### ISSUE TITLE
Crashes with latest Quilt on 1.20.1

### ISSUE THREAD

**Issue by tmprnc** [FrozenBlock/WilderWild#313]
**Description of the Bug Encountered**
The game crashes using the latest version of the mod and the latest version of functionally all non-Fabric versions of APIs it could need. It requires a future version of Fabric which Quilt doesn't match yet.

**Steps To Reproduce**
1. Install latest Quilt build (0.21.1-beta 2)
2. Install Quilted FAPI (latest version according to Modrinth)
3. Install Wilder Wilds
4. Game crashes, requiring a version of Fabric loader that Quilt does not match

**Expected Behavior**

The game starting.

**Screenshots**

![1697818616](https://github.com/FrozenBlock/WilderWild/assets/3785923/d360113b-d7db-4f67-b09e-2184ddb81cd3)

**Logs**

```
[18:15:21] [main/INFO]: Loading Minecraft 1.20.1 with Quilt Loader 0.21.1-beta.2
[18:15:21] [main/INFO]: Loading mappings: jar:file:/C:/Users/Temperance/AppData/Roaming/PrismLauncher/libraries/net/fabricmc/intermediary/1.20.1/intermediary-1.20.1.jar!/mappings/mappings.tiny
[18:15:24] [main/WARN]: Warnings for mod "dungeons_arise"'s mod json:
- Unsupported root entry "credits" at  at line 12 column 12 path $.credits
[18:15:26] [main/WARN]: Unknown file in mods folder: <mods>/YungsBetterJungleTemples-1.20-Forge-2.0.jar
Port = 5975
[18:15:27] [main/ERROR]: Crashed! The full crash report has been saved to C:\Users\Temperance\AppData\Roaming\PrismLauncher\instances\1.20\.minecraft\crash-reports\crash-2023-10-20_18.15.27.1203-quilt_loader.txt
---- Quilt Loader: Failed to load ----
Date/Time: 2023/10/20 18:15:27.1313
Quilt Loader Version: 0.21.1-beta.2
-- Error 1 --
Wilder Wild requires version [0.14.23, ?) of fabricloader, which is missing!
- <mods>/WilderWild-2.1.10-Fabric+1.20.1.jar
-- Plugin State --
Cycle number = 1
Cycle Step = SOLVE
Loaded Plugins (2):
 - 'quilt_loader' (Builtin)
 - 'quilted_fabric_loader' (Builtin)
---- end of report ----
[18:15:27] [main/ERROR]: For more details see the full crash report file: C:\Users\Temperance\AppData\Roaming\PrismLauncher\instances\1.20\.minecraft\crash-reports\crash-2023-10-20_18.15.27.1203-quilt_loader.txt
```

**Additional context**

**Comment by AViewFromTheTop** [comment]
@Treetrain1 can we just remove the fabric loader requirement lol
It’s kinda dumb tbh considering we only need to worry about FAPI which will have its own requirement anyway (I think)

**Comment by Treetrain1** [comment]
Resolved in 2.1.11
